src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -7833,19 +7833,17 @@ bool Unit::HandleDummyAuraProc(Unit* victim, uint32 damage, AuraEffect* triggere
 
                             triggered_spell_id = 31803;
 
-                            // On target with 5 stacks of Holy Vengeance direct damage is done
                             if (Aura* aur = victim->GetAura(triggered_spell_id, GetGUID()))
                             {
                                 if (aur->GetStackAmount() == 5)
                                 {
                                     if (stacker)
                                         aur->RefreshDuration();
-
-                                    CastSpell(victim, 42463, true, castItem, triggeredByAura);
-                                    return true;
                                 }
                             }
 
+                            CastSpell(victim, 42463, true, castItem, triggeredByAura);
```

Comment:
you move it you fix it. spell id

Reply:
Ain't no way you wanted me to set up enums in Unit.cpp for specific spells, so I added comments for the spell names. I hope that's what you wanted.

Reply:
Yes